site2/docs/reference-metrics.md

```diff
@@ -220,6 +221,7 @@ All the topic metrics are labelled with the following labels:
 | pulsar_publish_rate_limit_times | Gauge | The number of times the publish rate limit is triggered. |
 | pulsar_throughput_in | Gauge | The total throughput of the topic coming into this broker (byte per second). |
 | pulsar_throughput_out | Gauge | The total throughput of the topic going out from this broker (byte per second). |
+| pulsar_consumer_msg_ack_rate | Gauge | The total message acknowledge rate of the topic coming into this broker  (message per second). |
```

Comment:
```suggestion
| pulsar_consumer_msg_ack_rate | Gauge | The total message acknowledgment rate of the topic coming into this broker  (message per second). |
```

Reply:
Is it a rate of a namespace or a topic? The subject is different between Line175 and Line224.

Reply:
OK. My suggestion is to use "connected to" rather than "coming into", following the description of `pulsar_consumers_count`.